### ISSUE TITLE
Make all widgets and lists use dates consistently

### ISSUE THREAD

**Issue by bleroy** [bleroy/OrchardImportTest#1189]
Archives use published, the main blog list uses created.  

Work Item Details
--------------------
**Original CodePlex Issue:**	[Issue 17297](http://orchard.codeplex.com/workitem/17297)
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	andrerod
**Reported on:**	Feb 2, 2011 at 11:29 PM
**Reported by:**	BertrandLeRoy
**Updated on:**	Jun 28, 2011 at 12:14 AM
**Updated by:**	samward
**Closed on:**	May 31, 2011 at 9:05 PM
**Closed by:**	suhacan
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	andrerod
**Reported on:**	Feb 2, 2011 at 11:29 PM
**Reported by:**	BertrandLeRoy
**Updated on:**	Jun 28, 2011 at 12:14 AM
**Updated by:**	samward
**Closed on:**	May 31, 2011 at 9:05 PM
**Closed by:**	suhacan

**Comment by bleroy** [comment]
On *2011-02-03 07:31:38 UTC*, **@BertrandLeRoy** commented:

See these duplicates for details: 
http://orchard.codeplex.com/workitem/17294 
http://orchard.codeplex.com/workitem/17296

**Comment by bleroy** [comment]
On *2011-02-09 02:52:53 UTC*, **@BertrandLeRoy** commented:

When looking at this, dear developer, please touch-base with me and I'll do an audit of date usage throughout the codebase.

**Comment by bleroy** [comment]
On *2011-03-07 23:52:27 UTC*, **@rohancragg** commented:

In the meantime (while this is made consistent), can anyone advise how to alter a view or can I add something to a theme to make the blog (and other custom lists) order on my chosen field(s)?

**Comment by bleroy** [comment]
On *2011-03-08 01:18:34 UTC*, **@rohancragg** commented:

(also posted here: 
http://stackoverflow.com/questions/5223279/sorting-and-filtering-lists-in-orchard)

**Comment by bleroy** [comment]
On *2011-03-08 08:58:02 UTC*, **@BertrandLeRoy** commented:

For your custom lists, you can order on anything you want, that's part of your query.

**Comment by bleroy** [comment]
On *2011-04-27 01:42:12 UTC*, **@BertrandLeRoy** commented:

André, please talk to me when you look at this.

**Comment by bleroy** [comment]
On *2011-04-27 01:46:32 UTC*, **@BertrandLeRoy** commented:

Add doc comments explaining proper use of those dates, too.
We should use creation date throughout the UI. Related bug: 
http://orchard.codeplex.com/workitem/17772 which is to make creation date editable.

**Comment by bleroy** [comment]
On *2011-05-27 03:09:16 UTC*, **@andrerod** commented:

I believe this was arleady fixed by 17825 and 17772.

**Comment by bleroy** [comment]
On *2011-06-27 20:44:01 UTC*, **@samward** commented:

I'm not sure if I have missed something, but it seems that the Archives are still using the published date, while the main blog list is using created. Can anyone else confirm what I am seeing / what I should be seeing??
This was using the v1.2 Azure package drop (from the downloads page).

**Comment by bleroy** [comment]
On *2011-06-28 04:35:42 UTC*, **@BertrandLeRoy** commented:

@Sam: might you have some old archive data in there?

**Comment by bleroy** [comment]
On *2011-06-28 07:14:32 UTC*, **@samward** commented:

@BertrandLeRoy hrm now I feel silly. I did manually rebuild the Archive Part database records, but when I tried to view part of the archive (i.e. /blog/Archive/2011/06) I was getting inconsistant results. But now that I have checked again this morning,
 they seem to be correct. Maybe there was some caching that was causing me to see the wrong results.....

Sorry about that.

Cheers
